src/chainparams.cpp

```diff
@@ -192,7 +192,8 @@ class CMainParams : public CChainParams {
         consensus.DIP0024Height = 1737792; // 0000000000000001342be9c0b75ad40c276beaad91616423c4d9cb101b3db438
         consensus.DIP0024QuorumsHeight = 1738698; // 000000000000001aa25181e4c466e593992c98f9eb21c69ee757b8bb0af50244
         consensus.V19Height = 1899072; // 0000000000000015e32e73052d663626327004c81c5c22cb8b42c361015c0eae
-        consensus.MinBIP9WarningHeight = 1899072 + 2016; // V19 activation height + miner confirmation window
+        consensus.V20Height = 1987776; // 000000000000001bf41cff06b76780050682ca29e61a91c391893d4745579777
```

Comment:
what's the minimum set of commits needed in this PR to merge in 47def13162c553bcc7de4de396fa15a91b0727da?

Reply:
two commits can be excluded from this PR as just useful follow-ups:
 - c15b3d9a3f51fc3395d23348c8e69b30af4cd8f7 (feat: activate v20 on the same block as v19)
 - 4ade970884e9ad85f20b67b3902a6934490580ed (feat: speedup functional test feature_governance.py by generating less blocks)

all others are required for 47def13162c553bcc7de4de396fa15a91b0727da